requirements/testing/travis_flake8.txt

```diff
@@ -1,4 +1,3 @@
 # Extra pip requirements for the travis flake8 build
 
 flake8
```

Comment:
Does this need pinning to >=3.7 now?

Reply:
Probably. However, it's not working as expected. In the CI run here, per-file-ignores from the config are not respected. Needs further investigation.

I'm pinning to <3.7 for now to restore a working copy of flake8.